src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/message/MergeService.kt

```diff
@@ -72,7 +72,6 @@ class MergeService(
   private fun handleSourceUuidWithSingleRecord(mergeEvent: MergeEvent, sourcePersonEntity: PersonEntity, targetPersonEntity: PersonEntity) {
     mergeRecord(mergeEvent, sourcePersonEntity, targetPersonEntity) { sourcePerson, targetPerson ->
       linkSourceUuidToTargetAndMarkAsMerged(sourcePerson!!, targetPerson)
-      sourcePerson.personKey?.let { personKeyRepository.save(it) }
```

Comment:
Not needed as saved in method above (duplication) causing consistent `row updated or deleted`